core/iwasm/libraries/libc-wasi/sandboxed-system-primitives/include/wasmtime_ssp.h

```diff
@@ -22,1347 +18,594 @@
 #include <stddef.h>
 #include <stdint.h>
 
+#include "platform_wasi.h"
```

Comment:
Had better include "bh_patform.h", we usually include it instead of including files under core/shared/platform/include and core/shared/utils.

Reply:
changed to `bh_platform.h`, thanks